packages/generator-single-spa/src/root-config/templates/root-config.package.json

```diff
@@ -25,6 +25,7 @@
     "eslint-config-important-stuff": "^1.1.0",
     "eslint-config-prettier": "^7.0.0",
     "eslint-plugin-prettier": "^3.2.0",
+    "html-webpack-plugin": "^5.3.1",
```

Comment:
Good catch! html-webpack-plugin is a dependency of webpack-config-single-spa but we're turning off html generation from webpack-config-single-spa and requiring html-webpack-plugin directly inside of the root config webpack config. So this should be a dependency of root configs 👍🏿